src/ex1/script_ex1.js

```diff
@@ -0,0 +1,141 @@
+const addButton = document.querySelector("#add-task-button");
+const triangle = document.querySelector("#triangle");
+const box = document.querySelector("#box");
+const welcome = document.querySelector("#welcome-sentence");
+const addTaskButton = document.querySelector(".task-button");
+const addTaskInput = document.querySelector(".task-input");
+const clearAllButton = document.querySelector("#clear-all-button");
+const sortByNameButton = document.querySelector("#sort-by-name-button");
+const listOfTasks = [];
+
+addTaskButton.addEventListener("click", addTask);
+clearAllButton.addEventListener("click", clearAll);
+sortByNameButton.addEventListener("click", sortByName);
+
+addTaskInput.addEventListener("keypress", (event) => {
+  if (event.key === "Enter") {
+    addTask();
+  }
+});
+
+function addTask() {
+  const task = document.querySelector(".task-input").value;
+  const text = capitalize(task);
+
+  if (text === "") {
+    alert("Input cannot be empty");
+  } else {
+    if (listOfTasks.length === 0) {
+      hideWelcomePart();
+    }
+    listOfTasks.push(text);
+    createTask(text);
+  }
+}
+
+function hideWelcomePart() {
+  addButton.classList.remove("hithere");
+  clearAllButton.classList.remove("hidden");
```

Comment:
Moving the welcome page into its own container (maybe with a shared class for some of the styles) will make your life easier here - 1 line of code instead of 6. Also in the current state, every new element you'll add to the task-list container will require you to handle it in js as well